ckan/model/activity.py

```diff
@@ -158,7 +177,27 @@ def _activities_at_offset(q: QActivity,
     '''
     Return a list of all activities at an offset with a limit.
     '''
-    return _activities_limit(q, limit, offset).all()
+    return _activities_limit(q, limit, offset=offset).all()
+
+
+def _activities_after(q: QActivity,
+                      limit: int,
+                      after: datetime.datetime
+    ) -> list[Activity]:
+    '''
+    Return a list of all activities after a timestamp.
+    '''
+    return _activities_limit(q, limit, after=after).all()
+
+
+def _activities_before(q: QActivity,
+                      limit: int,
+                      before: datetime.datetime
+    ) -> list[Activity]:
+    '''
+    Return a list of all activities before a timestamp.
+    '''
+    return _activities_limit(q, limit, before=before).all()
```

Comment:
what do we need these 1-line functions for?

Reply:
They are useful to reuse between packages, users, group for activities. I delete the 2 I create.